src/components/forms/FormComponentBase/index.ts

```diff
@@ -162,7 +162,7 @@ export class FormComponentBase extends BridgeBase {
 
     // Always push a ValidateMessageValidator to the set of validators. This is how
     //  we get messages from "validatemessage" into the Lion validation process.
-    this.lionValidatorsArray.push(new ValidateMessageValidator());
+    this.lionValidatorsArray.push(new ValidateMessageValidator(this));
```

Comment:
So why are we passing **this** to the constructor of **ValidateMessageValidator**.  How does the type **param** align with instances of classes that extend FormComponentBase?  Basically I don't understand the @lion/ui/form-core Validator I think.

Reply:
The custom validator is triggered when we enter something in field, we determine the validation is failed based on validateMessage prop in pconn. so to check whether validation has failed or not ( this.param?.getComponentProp('validatemessage') ) we require to pass this.